tools/esdlcmd/esdlcmd_shell.cpp

```diff
@@ -180,14 +180,15 @@ bool EsdlCMDShell::parseCommandLineOptions(ArgvIterator &iter)
 void EsdlCMDShell::usage()
 {
     fprintf(stdout,"\nUsage:\n"
-        "    esdl [--version] <command> [<args>]\n\n"
+        "    esdl <command> [<args>]\n\n"
            "Commonly used commands:\n"
            "   xml               Generate XML from ESDL definition.\n"
            "   ecl               Generate ECL from ESDL definition.\n"
            "   xsd               Generate XSD from ESDL definition.\n"
            "   wsdl              Generate WSDL from ESDL definition.\n"
            "   publish           Publish ESDL Definition for ESP use.\n"
            "   list-definitions  List all ESDL definitions.\n"
+           "   get-definition    Get ESDL definition.\n"
```

Comment:
@rpastrana get-definition command is already there, it was only missing from the usage.